db/view/view.cc

```diff
@@ -378,24 +383,6 @@ class view_filter_checking_visitor {
     }
 };
 
-static query::partition_slice make_partition_slice(const schema& s) {
-    query::partition_slice::option_set opts;
-    opts.set(query::partition_slice::option::send_partition_key);
-    opts.set(query::partition_slice::option::send_clustering_key);
-    opts.set(query::partition_slice::option::send_timestamp);
-    opts.set(query::partition_slice::option::send_ttl);
-    opts.set(query::partition_slice::option::always_return_static_content);
-    return query::partition_slice(
-            {query::full_clustering_range},
-            s.static_columns()
-                    | std::views::transform(std::mem_fn(&column_definition::id))
-                    | std::ranges::to<query::column_id_vector>(),
-            s.regular_columns()
-                    | std::views::transform(std::mem_fn(&column_definition::id))
-                    | std::ranges::to<query::column_id_vector>(),
-            std::move(opts));
-}
```

Comment:
`make_partition_slice` and `flush_base` have been introduced in one commit and then moved in another one (`db/view: introduce view_building_worker`). Please fix this.

Reply:
fixed